pymatviz/colormaps.py

```diff
@@ -11,18 +11,18 @@
 def combine_two(
     cmaps: list[Colormap | str],
     node: float = 0.5,
-    N: int = 256,
```

Comment:
In fact this terrible naming is taken from the official `matplotlib` [doc](https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.Colormap.html) here, just to be consistent with the upstream `matplotlib`, I personally would be more than happy to switch to a more meaningful name as this one.

> ```python
> def __init__(self, name, N=256):
>     """
>     Parameters
>     ----------
>     name : str
>         The name of the colormap.
>     N : int
>         The number of RGB quantization levels.
>     """
> ```